packages/shared/lib/core/profile-manager/api/events/newOutput.ts

```diff
@@ -23,7 +23,7 @@ export async function handleNewOutputEvent(accountId: string, event: NewOutputEv
         syncBalance(account.id)
         addActivityToAccountActivitiesInAllAccountActivities(
             account.id,
-            await new Activity().setFromOutputData(event.output, account)
+            await new Activity().setFromOutputData(event.output, account, event.transactionInputs)
```

Comment:
We also optionally get the transaction so shouldn't we just call Activity().setFromTransaction if its available?

Reply:
the transaction we get isn't like the transactions we use in `setFromTransaction`. These `ITransactionPayload`s contain a lot less information than the `Transaction`s, so for now, the only additional information we can use are from the inputs